lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2178,9 +2206,13 @@ LogicalResult StmtEmitter::visitStmt(InstanceOp op) {
   // name, then use it here.  This is a hack because we lack proper support for
   // parameterized modules in the RTL dialect.
   if (auto extMod = dyn_cast<RTLModuleExternOp>(moduleOp)) {
-    indent() << extMod.getVerilogModuleName();
-  } else {
-    indent() << op.moduleName();
+    auto verilogName = extMod.verilogNameAttr();
+    if (!verilogName)
+      verilogName = extMod->getAttrOfType<StringAttr>(
+          ::mlir::SymbolTable::getSymbolAttrName());
+    indent() << emitter.getModuleName(verilogName);
```

Comment:
Plz add a `getNameAttr()` method to RTLModuleExternOp like you did for Module

Reply:
Added ```StringAttr getVerilogModuleNameAttr()``` to ```RTLModuleExternOp```.